src/borg/archiver.py

```diff
@@ -2179,8 +2191,8 @@ def do_break_lock(self, args, repository):
             /home/*/junk
             *.tmp
             fm:aa:something/*
-            re:^/home/[^/]\\.tmp/
-            sh:/home/*/.thumbnails
+            re:^home/[^/]\\.tmp/
+            sh:home/*/.thumbnails
```

Comment:
same here!?

Reply:
I removed the / to show that it can be used or be left out.

Reply:
ok